.travis.yml

```diff
@@ -1,17 +1,21 @@
 language: node_js
 node_js:
-- 6.0
+- 7
 notifications:
   email: false
 sudo: false
 before_install:
-  openssl aes-256-cbc -K $encrypted_ca4a154c2e6b_key -iv $encrypted_ca4a154c2e6b_iv -in config/tenon.json.enc -out config/tenon.json -d
+  - if [[ `npm -v` != 5* ]]; then npm i -g npm@latest; fi
```

Comment:
Sorry, didn't read the conditional when I suggested the `@latest` earlier; to make sure we don't reinstall `@latest` constantly when `npm@6` comes out we can do:

```bash
- if [[ `npm -v | cut -d. -f1` -lt 5 ]]; then npm i -g npm@latest; fi
```

`cut` will split the version number by the `.` delimeter and then give us the first field, which is the major number in the version. If this is less than 5, install the latest version of `npm`.